<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>applusplus</author>
 <class>APInjectorClass</class>
 <widget class="QMainWindow" name="APInjectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AP Injector 1.5 :: by applusplus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="apinjector.qrc">
    <normaloff>:/injection.ico</normaloff>:/injection.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabInjector">
     <attribute name="title">
      <string>Injection</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxDll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>72</y>
        <width>390</width>
        <height>45</height>
       </rect>
      </property>
      <property name="title">
       <string>Dynamic Link Library</string>
      </property>
      <widget class="QLabel" name="labelDllPath">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>18</y>
         <width>25</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxDllPath">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>16</y>
         <width>280</width>
         <height>22</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhPreferNumbers</set>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="butDllSearchPath">
       <property name="geometry">
        <rect>
         <x>322</x>
         <y>16</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxInjection">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>390</width>
        <height>65</height>
       </rect>
      </property>
      <property name="title">
       <string>Injection</string>
      </property>
      <widget class="QLabel" name="labelInjectionMethod">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>18</y>
         <width>45</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Method:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbInjectionMethodAuto">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>18</y>
         <width>75</width>
         <height>17</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Waiting x seconds befor starting the injection</string>
       </property>
       <property name="text">
        <string>Auto delay</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbInjectionMethodInstant">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>18</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Inject immediate</string>
       </property>
       <property name="text">
        <string>Instant</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsbInjectionDelay">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>15</y>
         <width>55</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Delay time in seconds</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>3600.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="labelInjectionDelaySec">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>18</y>
         <width>25</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sec.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="butInjection">
       <property name="geometry">
        <rect>
         <x>285</x>
         <y>15</y>
         <width>100</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Start injection with chosen options</string>
       </property>
       <property name="text">
        <string>Start Injection</string>
       </property>
      </widget>
      <widget class="QLabel" name="labInjectionStatus">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>43</y>
         <width>375</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Standby</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxProcess">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>2</y>
        <width>390</width>
        <height>68</height>
       </rect>
      </property>
      <property name="title">
       <string>Process</string>
      </property>
      <widget class="QLabel" name="labelPID">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>18</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxPID">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>15</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelPName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>18</y>
         <width>35</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxPName">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>15</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhPreferNumbers</set>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="butAddNewProcesses">
       <property name="geometry">
        <rect>
         <x>354</x>
         <y>15</y>
         <width>30</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>P</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbProcess">
       <property name="geometry">
        <rect>
         <x>88</x>
         <y>45</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Seaching for a process (e.g. game.exe)</string>
       </property>
       <property name="text">
        <string>Process</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbApplication">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>45</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Searching for a window name (e.g. Counter-Strike Source)</string>
       </property>
       <property name="text">
        <string>Application</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelSearchMethod">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>45</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Search method:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxEject">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>188</y>
        <width>391</width>
        <height>42</height>
       </rect>
      </property>
      <property name="title">
       <string>Eject</string>
      </property>
      <widget class="QPushButton" name="butEject">
       <property name="geometry">
        <rect>
         <x>285</x>
         <y>12</y>
         <width>100</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Eject a injected DLL</string>
       </property>
       <property name="text">
        <string>Eject</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelEjectNote">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>18</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Free a loaded/injected DLL:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelCopyright">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>270</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);</string>
    </property>
    <property name="text">
     <string>applusplus © 2013-2014</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionClose_after_injection"/>
    <addaction name="actionTray_on_minimize"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionClose_after_injection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close after injection</string>
   </property>
  </action>
  <action name="actionTray_on_minimize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="apinjector.qrc"/>
 </resources>
 <connections/>
</ui>
